src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__gradebook_audit.sql

```diff
@@ -0,0 +1,466 @@
+                true,
+                false
+            ) as qt_grade_70_comment_missing,
+
+            case
+                when
+                    region = 'Miami'
+                    and is_quarter_end_date_range
+                    and quarter_comment_value is null
+                then true
+                when
+                    region != 'Miami'
+                    and is_quarter_end_date_range
+                    and school_level = 'ES'
+                    and (course_name = 'HR' or credit_type in ('MATH', 'ENG'))
+                    and quarter_comment_value is null
+                then true
+                else false
+            end as qt_comment_missing,
+
+            if(
+                quarter_course_percent_grade_that_matters > 100, true, false
+            ) as qt_percent_grade_greater_100,
+
+            if(
+                percent_graded_completion_by_cat_qt_audit_week != 1
+                and region_school_level not in ('ESCamden', 'ESNEwark'),
+                true,
+                false
+            ) as percent_graded_completion_by_qt_audit_week_not_100,
+
+            if(
+                grade_level > 4
+                and ada_above_or_at_80 = 1
+                and quarter_course_grade_points_that_matters < 2.0,
+                true,
+                false
+            ) as qt_student_is_ada_80_plus_gpa_less_2,
+
+            if(
+                grade_level > 4
+                and abs(
+                    round(category_quarter_average_all_courses, 2)
+                    - round(category_quarter_percent_grade, 2)
+                )
+                >= 30,
+                true,
+                false
+            ) as grade_inflation,
+
+            if(
+                region = 'Miami' and category_quarter_percent_grade is null, true, false
+            ) as qt_category_grade_missing,
+
+            case
+                when isexempt = 1
+                then false
+                when
+                    school_level = 'MS'
+                    and assign_final_score_percent
+                    not in (50, 55, 58, 60, 65, 68, 70, 75, 78, 80, 85, 88, 90, 95, 100)
+                then true
+                when
+                    school_level = 'HS'
+                    and assign_final_score_percent
+                    not in (50, 55, 58, 60, 65, 68, 70, 75, 78, 80, 85, 88, 93, 97, 100)
+                then true
+                else false
+            end as assign_score_not_conversion_chart_options,
+
+            if(
+                region_school_level not in ('CamdenES', 'NewarkES')
+                and sum(
+                    if(assignmentid is null and date_enrolled >= duedate - 7, 0, 1)
+                ) over (
+                    partition by
+                        _dbt_source_relation, sectionid, student_number, `quarter`
+                )
+                = 0,
+                true,
+                false
+            ) as qt_assign_no_course_assignments,

```

Comment:
what should this audit check for? student scores or the presence of any assignments?